app/controllers/api_docs/pages_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module ApiDocs
+  class PagesController < ApiDocs::BaseController
+    def home; end
```

Comment:
This is probably fine for now but I'm a little weary of having a method per page because it means this controller could end up growing with a bunch of empty methods. Would be good to aim for maybe a show action which would look up the page name from the directory based on the params eg `api-docs/:page` and then use `params[:page`